drivers/hmis_external_apis/app/jobs/hmis_external_apis/ac_hmis/data_warehouse_upload_job.rb

```diff
@@ -125,6 +126,30 @@ def posting_export
       uploader.run!
     end
 
+    def custom_fields_export
+      cded_export = HmisExternalApis::AcHmis::Exporters::CdedExport.new
+      cded_export.run!
+
+      cde_export = HmisExternalApis::AcHmis::Exporters::CdeExport.new
+      cde_export.run!
+
+      uploader = Exporters::DataWarehouseUploader.new(
+        filename_format: '%Y-%m-%d-custom-fields.zip',
```

Comment:
Outstanding q on PT ticket to verify these client-facing filenames

Reply:
These look good to me. (We usually just decide the names and let them know).